sandbox/modules/identity/src/main/java/org/opensearch/identity/configuration/Hideable.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.identity.configuration;
+
+public interface Hideable {
```

Comment:
Let's add these only when needed. There is no concept of hidden or reserved at the moment.